tests/rules/no-exposed-subjects.ts

```diff
@@ -0,0 +1,375 @@
+/**
+ * @license Use of this source code is governed by an MIT-style license that
+ * can be found in the LICENSE file at https://github.com/cartant/eslint-plugin-rxjs
+ */
+
+import { stripIndent } from "common-tags";
+import rule = require("../../source/rules/no-exposed-subjects");
+import { ruleTester } from "../utils";
+
+ruleTester({ types: true }).run("no-exposed-subjects", rule, {
+  valid: [
+    stripIndent`
+      import { Subject } from 'rxjs';
+
+      const variable = new Subject<void>();
+    `,
+    stripIndent`
+      import { Subject } from 'rxjs';
+
+      function foo(a$: Subject<any>): Subject<any> {
+        return a$;
+      }
+    `,
+    stripIndent`
+      import { Observable, Subject } from 'rxjs';
+
+      class Mock {
+        private a = new Subject<void>();
+        private readonly b = new Subject<void>();
+        private c: number;
+
+        constructor(
+          private k: Subject<any>,
+          private m: Observable<any>,
+          z: Subject<any>,
```

Comment:
Should this be an error (as in rxjs-tslint-rules)?

Reply:
No, I don't think it should be an error. Only `public` or `protected` member parameters should be.